README.rst

```diff
@@ -3,4 +3,78 @@ Aexpect: Control your interactive applications
 
 This module provides services similar to the `pexpect` python library,
 so to speak, spawn and control interactive applications, such as `ssh`,
-`sftp` and others.
\ No newline at end of file
+`sftp` and others although it can be useful for pure streams as well.
+
+It's enhanced of multi-pattern matching, convenience features for
+not-only-linux terminals as well as support for terminals that "spit"
+extra output from time to time (eg. kernel messages) over the output.
+
+There are also extra classes to simplify executing parts of program
+on a different location (distributed programming) or writing controls
+executed on another host (metaprogramming).
+
+Simple usage
+------------
```

Comment:
Good idea about this, another alternative would be "Quick start" but both work well.